app/cronjob/trailingTrade/step/get-indicators.js

```diff
@@ -124,6 +123,8 @@ const calculateNextBestBuyAmount = (
       }
     );
 
+  const buyTrigger = 1 + (currentPrice - lastBuyPrice) / lastBuyPrice;
```

Comment:
@rando128 

I found when the `lastBuyPrice` is not configured, which will set as `null`, it returns `Infinity` for `buyTrigger`.

```
    currentPrice => 9000
    lastBuyPrice => null
    buyTrigger => Infinity
```

Is `Infinity` intended or it should be null?

Reply:
Good catch. It doesn't make sense to even try to calculate it in this case. Maybe we could condition nextBestBuyCalculation code with a `if(lastBuyPrice > 0)` statement on line 391

Reply:
Ok, good. Do you want to update, or do you want me to update? :)

Reply:
yay! I will check the tests and let's merge it.